EIPS/eip-draft_held_token.md

```diff
@@ -0,0 +1,228 @@
+---
+eip: <to be assigned>
+title: Held token standard
+description: Standard interface to query ownership and balance of held tokens
+author: Devin Conley (@devinaconley)
+discussions-to: https://ethereum-magicians.org/t/erc-standard-for-held-non-fungible-token-nfts-defi/7117
+status: Draft
+type: Standards Track
+category (*only required for Standards Track): ERC
+created: 2021-09-21
+requires (*optional): 20, 165, 721, 1155
+---
+
+## Abstract
+
+The proposed standard defines a lightweight interface to expose functional ownership and balances of held tokens. This standard is implemented by smart contracts which hold ERC20, ERC721, or ERC1155 and is intended to be consumed by both on-chain and off-chain systems that rely on ownership and balance verification.
+
+## Motivation
+
+As different areas of crypto (DeFi, NFTs, etc.) converge and composability improves, there will more commonly be a distinction between the actual owner (likely a contract) and the functional owner (likely a user) of a token. Currently, this results in a conflict between mechanisms that require token deposits and systems that rely on those tokens for ownership or balance verification.
+
+This proposal aims to address that conflict by providing a standard interface for token holders to expose ownership and balance information. This will allow users to participate in these DeFi mechanisms without giving up existing token utility. Overall, this would greatly increase interoperability across systems, benefiting both users and protocol developers.
+
+Example implementers of this ERC standard include
+- staking or farming contracts
+- lending pools
+- time lock or vesting vaults
+- fractionalized NFT contracts
+
+Example consumers of this ERC standard include
+- governance systems
+- gaming
+- PFP verification
+- art galleries or showcases
+- token based membership programs
+
+## Specification
+
+Smart contracts implementing the held token standard MUST also implement `ERC165` and return true when the corresponding interface ID is passed.
```

Comment:
You should specify the ERC-165 identifier number explicitly.

Reply:
fixed